dbt_project.yml

```diff
@@ -2,7 +2,7 @@ config-version: 2
 require-dbt-version: [">=1.3.0", "<2.0.0"]
 
 name: 'sap_source'
-version: '0.1.0'
+version: '0.1.1'
```

Comment:
We won't be cutting a release so these changes should be reverted back to v0.1.0